sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3.java

```diff
@@ -247,9 +253,44 @@ public String getNamespace() {
       return namespace;
     }
 
+    /**
+     * The composite {@link PTransform} for reading from datastore.
```

Comment:
Make valid javadoc.

`<p>` tag for new paragraphs
`<ol>` `<li>` etc for lists.